### ISSUE TITLE
sendFriendRequest can no longer be completed due to ROBLOX's update on adding a CAPTCHA to the function.

### ISSUE THREAD

**Issue by a1sop0s** [sentanos/roblox-js#61]
ROBLOX has pushed an update to stop users from being spammed with bots making you buy robux, etc.
That being an update that requires you to complete a CAPTCHA when attempting to add someone as your friend.
There's no work-around to it.

**Comment by LiamTownsley** [comment]
http://www.megatypers.com/ - Captcha farms.

**Comment by sbhadr** [comment]
@aiospos Correct. I've created a pull request to fix this. Thanks for reporting it.
